docs/changelog.rst

```diff
@@ -26,6 +26,17 @@ KSQL 5.3.0 includes new features, including:
 * ``CREATE STREAM`` and ``CREATE TABLE`` will now allow you to create the topic if it is missing.
   To do this, specify the ``PARTITIONS`` and optionally ``REPLICAS`` in the ``WITH`` clause.
 
+* ``CREATE STREAM AS SELECT ...`` and ``CREATE TABLE AS SELECT ...`` statements will use the source
+topic partitions and replica counts for the sink topic if the config properties are not set in ``WITH`` clause.
```

Comment:
Continuing the next line of the bullet item without the necessary leading whitespace will break the bullet list.

If you look at the rendered file (e.g. https://github.com/confluentinc/ksql/blob/9764f2e4e9f113e7f66c5ca8026672da1dc1f285/docs/changelog.rst), you will notice that this problem exists multiple times in the file.

Reply:
Ahhh, good point! Thanks @miguno , fixed it :)